shell/platform/fuchsia/flutter/engine.cc

```diff
@@ -620,6 +620,10 @@ void Engine::WarmupSkps(
       surface_producer.ProduceOffscreenSurface(size).release();
   if (!skp_warmup_surface) {
     FML_LOG(ERROR) << "Failed to create offscreen warmup surface";
+    // Tell client that zero shaders were warmed up because warmup failed.
+    if (completion_callback) {
```

Comment:
Not sure why the completion callback is an optional as it could just as well be initiated with `std::nullptr_t`. But ok.

Ideally, the check would be `completion_callback.has_value() && completion_callback.value()` to handle the aforementioned scenario. But, I am not familiar with these interfaces. So perhaps that is not possible.